spacemacs/packages.el

```diff
@@ -1677,14 +1677,19 @@ If ARG is non nil then `ag' and `pt' and ignored."
           (call-interactively (spacemacs//helm-projectile-do-search-find-tool
                                tools))))
 
+      (setq helm-projectile-sources-list '(helm-source-projectile-projects
+                                           helm-source-projectile-files-list
+                                           helm-source-recentf
+                                           helm-source-pp-bookmarks
+                                           helm-source-file-cache))
```

Comment:
You are putting CtrlPMixed under `SPC p p` and this is not desirable, maybe a binding under `SPC f` which is more general could fit but even then I don't like mixing `files` and `project files`.
We don't starve for key bindings, we have a lot of them available so we should use them, I prefer to have more specific key bindings that can fit with other key bindings more easily. This is in some sense the application of the unix philosophy or more generally to have something that does only one thing but that does it well.
For what I saw in this sketch with mixed sources, I don't like it and I don't think we can do a good UX with this approach. 
A good example is `SPC b b` where there is the list of open buffers as the default source and the list of recent file as a secondary source. I never use it for open a recent file, I prefer to use `SPC f r` directly and directly pick the top file I need, and this is an expected behavior because before doing the command I already know if I want a buffer or a recent file. But is some cases the secondary source may be helpful, for instance if I believed that a file was opened, but note that in this case I used `SPC b b` because I wanted to switch buffer, not open a recent file. Opening a recent file is wired to `SPC f r` in my brain and I prefer it like this instead of mixing concepts which brings some noise to the mnemonic. This is separation of concerns applied to key bindings.
In the case of switching project, I know I want to switch project and I want a UI that make the process easy and natural and this is not negotiable. The fallback source should be thought in the scope of switch project and should get in the way of the initial UI, for instance for `SPC p p` I could see the bookmarks fit in, but `SPC p p` remains `switch project` not `switch project or bookmarks`, to open a bookmark this is ` SPC h b` (which could be a better mnemonic). If the project is not in the source of projectile projects then it could be ok to also provide the bookmarks.